code/src/index.css

```diff
@@ -5,9 +5,157 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: linear-gradient(
+      to bottom,
+      rgba(0, 0, 0, 0.3),
+      rgba(0, 0, 0, 0.4)
+    ),
+    url("./background-image.jpg");
+  background-repeat: repeat;
+  background-attachment: fixed;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.send-thought {
+  border: 1px solid black;
+  width: 90%;
+  height: 135px;
+  margin: 30px 0;
+  padding: 10px;
+  box-shadow: 5px 5px 1px black;
+  background-color: rgb(240, 236, 236);
+  position: relative;
+}
+
+.send-thought form {
+  display: flex;
+  flex-direction: column;
+}
+
+.send-thought textarea {
+  resize: none;
+  margin: 10px 0;
+}
+
+.send-thought-button {
+  width: 80%;
+  height: 36px;
+  border-radius: 20px;
+  border: none;
+  background-color: #ffadad;
+  font-weight: 600;
+  font-size: 12px;
+  position: absolute;
+  bottom: 10px;
+  left: 10px;
+}
+
+.send-thought-button:hover {
+  background-color: rgb(250, 123, 123);
+  cursor: pointer;
+}
+
+.send-heartemoji {
+  font-size: 10px;
+  margin: 0 3px;
+}
+
+.character-count {
+  position: absolute;
+  right: 10px;
+  bottom: 0px;
+  font-size: 12px;
+  font-weight: 650;
+}
+
+.red-length {
+  color: red;
+}
+
+.black-length {
+  color: #000000;
+}
+
+.thoughts-section {
+  width: 95%;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.thought-card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid black;
+  height: 135px;
+  margin-bottom: 30px;
+  padding: 10px;
+  box-shadow: 5px 5px 1px black;
+  word-break: break-word;
+  background-color: #ffffff;
+}
+
+.thought-text {
+  font-family: Courier New, monospace;
+}
+
+.time-text,
+.like-text {
+  font-size: 12px;
+  color: rgb(187, 186, 186);
+}
+
+.liked-heartemoji {
+  padding: 6px;
+  border-radius: 50%;
+  border: none;
+  margin-right: 5px;
+  cursor: pointer;
+}
+
+.top-of-card {
+  height: 70%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.bottom-of-card {
+  height: 30%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 5px;
+}
+
+@media (min-width: 768px) {
+  .main-container {
+    width: 390px;
+  }
+
+  .send-thought {
+    width: 358px;
+  }
+
+  .thoughts-section {
+    width: 380px;
```

Comment:
You could also have set a max-width on the first css rule, then you would not have needed to add media queries to style this 😃